tests/50-sd-verify.js

```diff
@@ -23,147 +22,158 @@ const {match} = endpoints.filterByTag({
 
 describe('ecdsa-sd-2023 (verify)', function() {
   describe('ecdsa-sd-2023 (verifiers)', function() {
-    let issuers;
-    let vcHolder;
-    before(async function() {
-      const {match: matchingIssuers} = endpoints.filterByTag({
-        tags: [...tags],
-        property: 'issuers'
-      });
-      const {match: matchingVcHolders} = endpoints.filterByTag({
-        tags: [...holderTags],
-        property: 'vcHolders'
-      });
-      // Uses 'Digital Bazaar' as default issuer to issue a verifiable
-      // credential for the verifier tests.
-      issuers = matchingIssuers.get(issuerName).endpoints;
-      const vcHolders = matchingVcHolders.get(holderName).endpoints;
-      vcHolder = vcHolders[0];
-    });
     // this will tell the report
     // to make an interop matrix with this suite
     this.matrix = true;
     this.report = true;
     this.rowLabel = 'Test Name';
     this.columnLabel = 'Verifier';
     this.implemented = [];
+    const testVectors = {
+      //signedCredentials
+      signed: [],
+      disclosed: {
+        //disclosedCredentials
+        base: [],
+        //nestedDisclosedCredentials
+        nested: [],
+        //disclosedDlCredentialNoIds
+        noIds: [],
+        array: {
+          //disclosedCredentialsWithFullArray
+          full: [],
+          //disclosedCredentialsWithLessThanFullSubArray
+          lessThanFull: [],
+          //disclosedCredentialsWithoutFirstArrayElement
+          missingElements: []
```

Comment:
What are these comments?

Reply:
they refer to the arrays that are actually used by the tests, not all implementers support each keyType so lessThanFull will have a map which theoretically could contain VCs issued with `p-256` and `p-384` which are then filtered before each implementer's test into an array containing only the vcs the implementer supports. TBH the way test data is created in the verify suites needs more thought and attention than this PR provides.

Reply:
hmm interesting - got it